### DISCUSSION TITLE
Add a new hotkey

### DISCUSSION THREAD

**Discussion by irviahvi** [M-Reimer/undoclosetab#134] [Ideas]
Is it possible to add the ability to assign a hotkey not only to open a closed tab, but also to open a list of closed tabs (as when right-clicking on the extension icon), thank you.

**Comment by M-Reimer** [comment]
After thinking a bit about this, I don't know how this could be added. The list itself is the context menu which is triggered using the right mouse button. And then you pick the tab to restore using the left mouse button.

So your hand has to be on the mouse anyway to left click the entry in the menu.

What exactly do you plan to optimize here? Maybe your problem can be solved with a different approach.

**Reply by irviahvi** [reply]
sometimes it's useful for me to just look at the list of closed tabs to see if I closed something by accident. This option (just opening a list of closed tabs using a hotkey) is implemented in many similar extensions (SimpleUndoRecents/SexyUndoCloseTab/etc) for Chrome, but I did not find any for Firefox :D

**Reply by M-Reimer** [reply]
Likely those are "menu only" extensions which don't use the context menu for the list.
That's not how my extension is built. One of the main goals was to be able to have both, the list and the "single click restore".
